quickOrder/database.py

```diff
@@ -0,0 +1,56 @@
+from .serializers import InventoryItemSerializer
+from .models import InventoryItem
+from .serializers import OrderSerializer
+from .models import Order
+from UserManagement.models import User
+
+def getAllInventoryObjects():
+    return InventoryItem.objects.all().order_by('id')
+
+def getInventoryObject(pk):
+    return InventoryItem.objects.get(id = pk)
+
+def updateInventoryStatus(updatedStatus, pk):
+    item = getInventoryObject(pk)
+    item.is_ordered = updatedStatus
```

Comment:
I think we removed is_ordered from the table. We should remove this function too?